enterprise/glassfish.common/src/org/netbeans/modules/glassfish/common/GlassfishInstance.java

```diff
@@ -716,8 +711,8 @@ public static void writeInstanceToFile(
     /** Configuration changes listener watching <code>domain.xml</code> file. */
     private final transient DomainXMLChangeListener domainXMLListener;
 
-    private transient InstanceContent ic;
-    private transient Lookup localLookup;
+    private final transient InstanceContent ic;
+    private final transient Lookup localLookup;
```

Comment:
`transient` will prevent that these fields are written to a file (maybe a glassfish config file?) or travel through the network (maybe when interacting with a remote glassfish instance?).

Maybe that was a design choice since there are multiple `private transient` fields (9). There are other `private final transient` fields, and this two fields can also became `final`. That's why a change it.

Reply:
A sorry, I was reading "transient", my mind was thinking "volatile". Forget it.